frontend/components/Accounts/Forms/ContactInput.tsx

```diff
@@ -82,6 +98,7 @@ const FieldInput = ({
             return;
         }
 
+        // bruh what is this
```

Comment:
Reminder to remove this - probably best to ask the question in a comment by highlighting a code block rather than leaving it in a comment just in case we accidentally miss it in review